app/assets/javascripts/custom.js

```diff
@@ -5,3 +5,7 @@
 //
 //
 
+//Auto hide for notification
+$(document).on('page:change', function(){
+     $(".notice-container").delay(5000).slideUp(500);
```

Comment:
Had to change from ".alert" to ".notice-container" for the notice to use the delay and slide up actions.  Using ".alert" didn't have any effect on my test instance.